client/src/config.ts

```diff
@@ -13,4 +13,7 @@ export const google = window.google;
 export const isWeb = window === window.parent;
 
 // Enable debugging on web by default
-export const isDebug = false || isWeb;
+export const isDebug = true;
+
+// Places API Key
+export const PLACES_KEY = "AIzaSyBkGOvn8gUfLrECXPQ5KFNCHUqoXr3mzCc";
```

Comment:
checking-in API KEY, this is should be an environment variable

Reply:
see #121 at this point imo securing client-side keys is a losing battle, we have mitigated risks, but will move to eliminate such usage in the future with #91 design